app/src/main/java/com/github/mik629/android/fundamentals/di/NetworkModule.kt

```diff
@@ -0,0 +1,68 @@
+package com.github.mik629.android.fundamentals.di
+
+import com.github.mik629.android.fundamentals.BuildConfig
+import com.github.mik629.android.fundamentals.data.network.ServerApi
+import com.squareup.moshi.Moshi
+import dagger.Module
+import dagger.Provides
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Response
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Module
+class NetworkModule {
+    @Provides
+    @Singleton
+    internal fun provideOkHttpClient(): OkHttpClient {
```

Comment:
what sense of internal?

Reply:
and why public if we can keep it internal? :)

Reply:
u can do class internal

keep in mind that partially internal class is something not normal
better split code by classes and make one of them internal

Reply:
Done, thanks